src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java

```diff
@@ -142,6 +280,11 @@ public void testModel() throws IOException {
         }
         Assert.assertNotNull(sim);
         Assert.assertFalse(errorInSimulator);
+
+        String testcase = path.substring(path.length() - 5);
+        sim.setStochasticSeed(stochasticSeeds[Integer.parseInt(testcase) - 1]);
+
+        ((SBMLNetwork) net).registerEvents(sim);
         // Initializes the observer for the amounts of molecule species
```

Comment:
What exactly do we initialize here? I see only an Object reference declaration and definition.

Reply:
Here, I am making the instance of the observer which keeps track of amounts and others. So added a comment `Initializes the observer ....`.

Reply:
> AmountIntervalObserver obs = null;
Where exactly are we making the instance of the observer here?

Reply:
https://github.com/hemilpanchiwala/SBSCL/blob/fern/src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java#L299

Reply:
In the line, you pointed out, no initialization happens. Only object reference declaration and definition.

The instance of the observer is created here:
https://github.com/draeger-lab/SBSCL/blob/41fb960ace6c6a34b99c4dcb307244c63b1a497d/src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java#L494

Probably this might help:
https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.9

Reply:
I thought that you wanted the line where the `AmountIntervalObserver obs = null;` was there. Removing the comment from there.